---
std_msgs/Float32 laser_max
std_msgs/Float32 laser_min
